addons/sale/models/sale_order_line.py

```diff
@@ -134,35 +134,20 @@ def _compute_price_reduce_taxexcl(self):
         for line in self:
             line.price_reduce_taxexcl = line.price_subtotal / line.product_uom_qty if line.product_uom_qty else 0.0
 
+    @api.depends('product_id')
```

Comment:
Why don't we add a depend on `order_id.fiscal_position_id` here ?

Reply:
You don't want any recomputation to happen on SOL level when anything is changed on the SO level.
It has always been like this and should stay like this.  We don't want a modification of the partner to trigger nearly all computes of the SOL (partner -> pricelist & fpos -> SOL prices, discounts -> amounts)